lib/sgov/sgov-adapter.spec.ts

```diff
@@ -0,0 +1,117 @@
+import {SgovAdapter} from "./sgov-adapter";
+import {httpFetch} from "../io/fetch/fetch-nodejs";
+import {PimClass} from "../pim/model";
+import {CimAdapter, IriProvider, PrefixIriProvider} from "../cim";
+import {FetchOptions} from "../io/fetch/fetch-api";
+import {CoreResourceReader} from "../core";
+
+let iriProvider: IriProvider;
+let adapter: CimAdapter;
+
+beforeAll(() => {
+  iriProvider = new PrefixIriProvider();
+  adapter = new SgovAdapter("https://slovník.gov.cz/sparql", httpFetch);
+  adapter.setIriProvider(iriProvider);
+});
+
+test("SgovAdapter.search()", async () => {
+  const query = "řidič";
+  const result = await adapter.search(query);
+  expect(result.map(cls => cls.pimHumanLabel?.cs)).toContain("Řidičský průkaz České republiky");
+}, 10 * 60 * 1000);
+
+describe("SgovAdapter.getClass()", () => {
+  test("existing class", async () => {
+    const query = "https://slovník.gov.cz/veřejný-sektor/pojem/fyzická-osoba";
+    const cls = await adapter.getClass(query) as PimClass;
+
+    expect(cls.pimHumanLabel?.en).toEqual("Natural Person");
+    expect(cls.pimHumanDescription?.en).toEqual("Natural Person is a human as a legal subject.");
+  }, 10 * 60 * 1000);
+
+  test("non existing class", async () => {
+    const query = "https://slovník.gov.cz/veřejný-sektor/pojem/křestní-jméno";
+    const cls = await adapter.getClass(query);
```

Comment:
I know class is a reserved keyword, but instead of using ```cls```, similar to ```itc``` .. I like to name this values ```actuial``` to emphasize that this is the actual value produced by the code. Then I would have ```expected``` variable with the expected state.